addons/mass_mailing/models/mailing.py

```diff
@@ -913,20 +913,30 @@ def _process_mass_mailing_queue(self):
                     # send the KPI mail only if it's the first sending
                     'kpi_mail_required': not mass_mailing.sent_date,
                 })
-
-        mailings = self.env['mailing.mailing'].search([
-            ('kpi_mail_required', '=', True),
-            ('state', '=', 'done'),
-            ('sent_date', '<=', fields.Datetime.now() - relativedelta(days=1)),
-            ('sent_date', '>=', fields.Datetime.now() - relativedelta(days=5)),
-        ])
-        if mailings:
-            mailings._action_send_statistics()
+        print("Mass mailing process executed")
+        if self.env['ir.config_parameter'].sudo().get_param('mass_mailing.mass_mailing_reports'):
```

Comment:
I would rather do the opposite, have a parameter to disable the feature, otherwise the feature will be disabled on each customers database (because you do not have a migration script to set it to True)